xml/requirements.txt

```diff
@@ -0,0 +1,5 @@
+certifi==2016.2.28
```

Comment:
Is there a reason we're not using the following?
certifi==2019.9.11
Before the requirements.txt file existed, just about an hour ago I installed a conda env with:
conda install -> Jinja2, lxml, PyYAML
and the rest of the versions were the same, and MarkupSafe/certifi were installed as dependencies.

Reply:
Thanks, `certifi` snuck in there, I will remove it